Improve the documentation of the "subdomain" field (#359)
okteto/docs: src/content/self-hosted/administration/configuration.mdx

### subdomain

> Okteto offers a wildcard subdomain feature that allows you to expose secure public endpoints for your development environments.

Reviewer: ```suggestion
Okteto's  [automatic SSL endpoints](ADD LINK) allows you to expose secure and unique public endpoints for your development environments. By default, all endpoints created by Okteto will use a combination of the name of the app, the namespace, and the `subdomain`.
```

> For instance, if you have a development environment named `app` in the `cindy` namespace, it will be accessible at https://app-cindy.example.com if `subdomain` is example.com.

Reviewer: ```suggestion
For example, if you have a development environment named `app` in the `cindy` namespace, it will be accessible at https://app-cindy.example.com if `subdomain` is example.com.
```

```yaml
subdomain: "example.com"
```

> The subdomain feature is also utilized to expose Okteto itself at https://okteto.$SUBDOMAIN.

Reviewer: ```suggestion
By default, Okteto's frontend and API services will be accessible via https://okteto.$SUBDOMAIN.
```

Once Okteto is installed, you can use `kubectl` to retrieve the public address of the Okteto NGINX Ingress Controller:

```console
kubectl get service -l=app.kubernetes.io/name=ingress-nginx,app.kubernetes.io/component=controller --namespace=okteto
```

The output will look something like this:

```
NAME                              TYPE           CLUSTER-IP   EXTERNAL-IP     PORT(S)                                     AGE
okteto-ingress-nginx-controller   LoadBalancer   10.0.7.73    34.68.230.234   80:30795/TCP,443:32481/TCP,1234:30885/TCP   5m
```

You'll need to take the `EXTERNAL-IP` address to create a DNS entry for `*.$SUBDOMAIN`.

> It's important to note that you can overwrite the domain where Okteto is exposed using the [publicOverride](self-hosted/administration/configuration.mdx#publicoverride) field.

Reviewer: ```suggestion
You can overwrite Okteto's public URL using the [publicOverride](self-hosted/administration/configuration.mdx#publicoverride) field.
```
